brigade/plugins/inventory/nsot.py

```diff
@@ -15,22 +15,38 @@ class NSOTInventory(Inventory):
         the name of the site the host belongs to.
 
     Environment Variables:
-        * ``NSOT_URL``: URL to nsot's API (defaults to ``http://localhost:8990/api``)
-        * ``NSOT_EMAIL``: email for authentication (defaults to admin@acme.com)
+        * ``nsot_url``: URL to nsot's API (defaults to ``http://localhost:8990/api``)
+        * ``nsot_email``: email for authentication (defaults to admin@acme.com)
+        * ``nsot_auth_method``: auth_header or auth_token
```

Comment:
wouldn't it be better to infer this? So if `nsot_secret_key` is specified use one method, otherwise use the other? Happy to leave it as it is, just suggesting a simplification

Reply:
So you mean just remove the nsot_auth_method parameter and rely only on nsot_secret_key is given or not?